### PR TITLE
[Storage] Support Blob inventory

### PR DESCRIPTION
**Pull request by blueww**
## Description

design pass review in: https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/924
The static analysis exception is caused by the cmdlets `New-AzStorageBlobInventoryPolicyRule` create a local object, don't need Should process.

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/924
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Storage/Storage.Management/help/Get-AzStorageBlobInventoryPolicy.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Storage.Management.dll-Help.xml
+Module Name: Az.Storage
+online version: https://docs.microsoft.com/en-us/powershell/module/az.storage/get-azstorageblobinventorypolicy
```
Comment (wyunchi-ms) [inline src/Storage/Storage.Management/help/Get-AzStorageBlobInventoryPolicy.md:4]: Neet to remove language `en-us`.

File: src/Storage/Storage.Management/help/New-AzStorageBlobInventoryPolicyRule.md [hunk 2]
```diff
@@ -0,0 +1,269 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Storage.Management.dll-Help.xml
+Module Name: Az.Storage
+online version: https://docs.microsoft.com/en-us/powershell/module/az.storage/new-azstorageblobinventorypolicyrule
```
Comment (wyunchi-ms) [inline src/Storage/Storage.Management/help/New-AzStorageBlobInventoryPolicyRule.md:4]: Neet to remove language `en-us`.

File: src/Storage/Storage.Management/help/Remove-AzStorageBlobInventoryPolicy.md [hunk 3]
```diff
@@ -0,0 +1,206 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Storage.Management.dll-Help.xml
+Module Name: Az.Storage
+online version: https://docs.microsoft.com/en-us/powershell/module/az.storage/remove-azstorageblobinventorypolicy
```
Comment (wyunchi-ms) [inline src/Storage/Storage.Management/help/Remove-AzStorageBlobInventoryPolicy.md:4]: Neet to remove language `en-us`.

File: src/Storage/Storage.Management/help/Set-AzStorageBlobInventoryPolicy.md [hunk 4]
```diff
@@ -0,0 +1,336 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Storage.Management.dll-Help.xml
+Module Name: Az.Storage
+online version: https://docs.microsoft.com/en-us/powershell/module/az.storage/set-azstorageblobinventorypolicy
```
Comment (wyunchi-ms) [inline src/Storage/Storage.Management/help/Set-AzStorageBlobInventoryPolicy.md:4]: Neet to remove language `en-us`.

File: src/Storage/Storage.Management/help/Get-AzStorageBlobInventoryPolicy.md (hunk 1 above)
Comment (blueww) [reply src/Storage/Storage.Management/help/Get-AzStorageBlobInventoryPolicy.md]: fixed

File: src/Storage/Storage.Management/help/New-AzStorageBlobInventoryPolicyRule.md (hunk 2 above)
Comment (blueww) [reply src/Storage/Storage.Management/help/New-AzStorageBlobInventoryPolicyRule.md]: fixed

File: src/Storage/Storage.Management/help/Remove-AzStorageBlobInventoryPolicy.md (hunk 3 above)
Comment (blueww) [reply src/Storage/Storage.Management/help/Remove-AzStorageBlobInventoryPolicy.md]: fixed

File: src/Storage/Storage.Management/help/Set-AzStorageBlobInventoryPolicy.md (hunk 4 above)
Comment (blueww) [reply src/Storage/Storage.Management/help/Set-AzStorageBlobInventoryPolicy.md]: fixed

**Review by wyunchi-ms — APPROVED** [review]

**Pull request merged**